test/git-open.bats

```diff
@@ -152,6 +152,26 @@ setup() {
 }
 
 
+@test "bitbucket: Bitbucket Server" {
+  # https://github.com/paulirish/git-open/issues/77#issuecomment-309044010
+  git remote set-url origin "https://user@mybb.domain.com/git/scm/ppp/rrr.git"
+  run ../git-open
+  assert_output "https://mybb.domain.com/git/projects/ppp/repos/rrr"
+}
+
+@test "bitbucket: Bitbucket Server branch" {
+  # https://github.com/paulirish/git-open/issues/80
+  git remote set-url origin "https://user@mybb.domain.com/git/scm/ppp/rrr.git"
+  git checkout -B "develop"
+  run ../git-open
+
+  # The following query args work with BB Server:
+  #     at=refs%2Fheads%2Fdevelop, at=develop, at=refs/heads/develop
+  # However /src/develop does not (unlike bitbucket.org)
+  assert_output "https://mybb.domain.com/git/projects/ppp/repos/rrr?at=develop"
```

Comment:
Minor change required here. If the `at` parameter is provided to select a branch, the `/browse` part is required after the repo name.

https://mybb.domain.com/git/projects/ppp/repos/rrr/browse?at=develop